recipes/orc/all/conandata.yml

```diff
@@ -2,8 +2,11 @@ sources:
   "2.0.0":
     url: "https://dlcdn.apache.org/orc/orc-2.0.0/orc-2.0.0.tar.gz"
     sha256: "9107730919c29eb39efaff1b9e36166634d1d4d9477e5fee76bfd6a8fec317df"
+  "1.9.3":
+    url: "https://dlcdn.apache.org/orc/orc-1.9.3/orc-1.9.3.tar.gz"
+    sha256: "f737d005d0c4deb65688ac3c0223ed530b0ba6258552555b2774dcdb77359b0f"
   "1.9.2":
-    url: "https://dlcdn.apache.org/orc/orc-1.9.2/orc-1.9.2.tar.gz"
+    url: "https://archive.apache.org/dist/orc/orc-1.9.2/orc-1.9.2.tar.gz"
```

Comment:
Any insight into why the URL has changed?

Reply:
AFAIK, only latest release of each major version resides at https://dlcdn.apache.org/orc/ and all older archives are moved to https://archive.apache.org/dist/orc/. I'm not sure if this is the convention of Apache projects. cc @dongjoon-hyun for more advise.

Reply:
BTW, it seems that we can get all version via https://archive.apache.org/dist/orc/.

Reply:
Let's then change it up so that we dont need to change revisions of old versions each time a new one comes out :)

Reply:
Fixed. It seems that I cannot retrigger CIs manually except to force push again.

Reply:
yes, no worries, the CI is failing due to unrelated reasons, we'll triggee it automatically after it's back online!

Reply:
Ya, you are right, @wgtmac . We need to use `https://archive.apache.org` to access all releases.

Reply:
Thanks!